maximbircu/devtools-library
# Add library API documentation

Closes: #40 


### What has been done
1. Cover the whole library with documentation, check [README](https://github.com/maximbircu/devtools-library/blob/40-add-api-documentation/README.md);
2. Add library extension Android samples;
3. Remove redundant `yml-` prefix;
4. Replace `defaultValue` with `default` everywhere to make the API consistent.

### How to test
Make sure the [library API docs](https://github.com/maximbircu/devtools-library/blob/40-add-api-documentation/README.md) are accessible, straight forward, and concise.
